README.md

```diff
@@ -1,14 +1,24 @@
-# Project
+<a href="https://marketplace.visualstudio.com/items?itemName=mindaro.mindaro" alt="Click to visit marketplace">
+    <img src="https://vsmarketplacebadge.apphb.com/version-short/mindaro.mindaro.svg">
+    <img src="https://vsmarketplacebadge.apphb.com/rating-star/mindaro.mindaro.svg">
+</a>
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+# VsCode Bridge to Kubernetes
 
-As the maintainer of this project, please make a few updates:
+> This repo has been populated for separating vscode extension implementation of [Bridge to Kubernetes](https://github.com/Azure/Bridge-To-Kubernetes).
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+Some important information reside here:
+
+* [How Bridge to Kubernetes works](https://docs.microsoft.com/en-us/visualstudio/bridge/overview-bridge-to-kubernetes)
+* [Use Bridge to Kubernetes (VS Code)](https://docs.microsoft.com/en-au/visualstudio/bridge/bridge-to-kubernetes-vs-code)
+
+## Telemetry
+
+This extension collects telemetry data to help us build a better experience for building applications with Azure Kubernetes Service and VS Code. We only collect the following data:
```

Comment:
Nit: Do we want to mention `Azure Kubernetes Service` specifically here, given that Bridge supports all Kubernetes clusters?